releng/org.eclipse.cdt.target/cdt-baseline.target

```diff
@@ -1,101 +1,47 @@
+      <unit id="org.eclipse.cdt.build.crossgcc.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.cmake.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.core.autotools.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.debug.dap.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.debug.dap.gdbjtag.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.debug.gdbjtag.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.debug.standalone.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.debug.ui.memory.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.docker.launcher.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.examples.dsf.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.gnu.multicorevisualizer.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.jsoncdb.arm" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.jsoncdb.hpenonstop" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.jsoncdb.intel" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.jsoncdb.microsoft" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.jsoncdb.nvidia" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.launch.remote.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.launch.serial.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.llvm.dsf.lldb.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.lsp.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.managedbuilder.llvm.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.meson.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.msw.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.qt.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.sdk.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.testsrunner.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.unittest.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.util" version="0.0.0"/>
+      <unit id="org.eclipse.cdt.visualizer.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.launchbar.remote.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.console.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.proxy.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.serial.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.telnet.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.connector.cdtserial.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.connector.remote.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.feature.feature.group" version="0.0.0"/>
+      <repository location="https://download.eclipse.org/tools/cdt/releases/latest"/>
```

Comment:
Usually we have repository at the top and then we sort locations by repository::location